contracts/hyperlane-mailbox/src/main.sw

```diff
@@ -5,7 +5,17 @@ use std::{auth::msg_sender, bytes::Bytes, call_frames::contract_id, logging::log
 use std_lib_extended::bytes::*;
```

Comment:
We also need to make the multisig ISM ownable & use `only_owner()` in a bunch of those functions, context: https://github.com/hyperlane-xyz/fuel-contracts/pull/16#discussion_r1157240430